SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1804,7 +1818,7 @@ void USpatialReceiver::OnCommandRequest(const Worker_Op& Op)
 	if (TargetObject == nullptr)
 	{
 		UE_LOG(LogSpatialReceiver, Warning, TEXT("No target object found for EntityId %d"), EntityId);
-		Sender->SendEmptyCommandResponse(ComponentId, CommandIndex, RequestId, Op.span_id);
+		Sender->SendEmptyCommandResponse(ComponentId, CommandIndex, RequestId, FSpatialGDKSpanId(Op.span_id));
```

Comment:
Is this safe is Op.span_id == nullptr, should FSpatialGDKSpanId handle that? What if tracing is disabled?

Reply:
I think this should handle nullptr and just become a null spanId. If et is enabled in the runtime I think they will always use a null spanId if a valid one is not provided, at least that is what I hear from Jon. I have updated FSpatialGDKSpanId to handle nullptr.